proposals/NNNN-Refactor-Fuel-Information-Related-Vehicle-Data.md

```diff
@@ -0,0 +1,159 @@
+# Refactor Fuel Information Related Vehicle Data
+
+* Proposal: [SDL-NNNN](NNNN-Refactor-Fuel-Information-Related-Vehicle-Data.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **In Review**
+* Impacted Platforms: [Core | HMI | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to consolidate Fuel Information related vehicle data so that it supports multiple type of propulsion sources used by vehicle.
```

Comment:
```suggestion
This proposal is to consolidate Fuel Information related vehicle data so that it supports multiple types of propulsion sources used by the vehicle.
```

Reply:
Done